src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -104,6 +104,11 @@ export default App.extend({
       this.toggleBulkSelect();
     });
 
+    this.listenTo(collectionView, 'click:patientSidebarButton', ({ model }) => {
```

Comment:
https://app.shortcut.com/roundingwell/story/29115/bug-patient-sidebar-sidebar-does-not-close-if-you-change-any-of-the-worklist-filter-settings is fixed by adding above:
```js
  onBeforeStart({ worklistId }) {
    if (this.isRestarting()) {
      Radio.request('sidebar', 'close');
```

Reply:
This did the trick. Included in the last commit. 

I also added a check to the Cypress test that updates a worklist filter while the sidebar is open. And verifies that the sidebar closes when that happens.